OrbitQt/orbittreeview.cpp

```diff
@@ -260,6 +248,27 @@ void OrbitTreeView::keyPressEvent(QKeyEvent* event) {
   }
 }
 
+void OrbitTreeView::selectionChanged(const QItemSelection& selected,
+                                     const QItemSelection& deselected) {
+  QTreeView::selectionChanged(selected, deselected);
+
+  // Dont trigger callbacks if selection was initiated internally.
+  if (is_internal_refresh) return;
+
+  // Row selection callback.
+  QModelIndexList rows = selectionModel()->selectedRows();
+  if (rows.size() == 1) {
```

Comment:
I believe this causes a bit of a weird behaviour because when selecting multiple items the `DataView` is not notified of the new selection. Before the `DataView` would still be notified only of one item, but it was always the last one to be selected (but only if with the mouse), while this changes it. I guess this is a broader problem but it's not ideal.

Reply:
there is currentIndex() in selection model. I think it should be used to call OnRowSelected regardless of number of selected rows.

Reply:
Thanks @dpallotti and @dimitry- , I agree it is better to notify on every selection change.  I implemented Dimitry's suggestion. In the sampling report, the callstack now always matches the currently selected function.

Reply:
is this happening when you use selectedIndex?

Reply:
What do you mean?  There is a selectedIndexes method, but no selectedIndex.

Reply:
sorry - yes, I meant currentIndex

Reply:
yep, using currentIndex, the callstack panel is always in sync with the current function.